src/Form/Checkbox.stories.js

```diff
@@ -12,6 +12,8 @@ export const Basic = () => <Checkbox />;
 
 export const Label = () => <Checkbox label="I'm a checkbox" disabled={boolean('Disabled', false)} />;
 
+export const isRadio = () => <Checkbox isRadio />;
```

Comment:
I'm sorry, super nit: 
```suggestion
export const Radio = () => <Checkbox isRadio />;
```

Reply:
😆 No worries!